cloudwatchlogs/cloudwatchlogs_lambda.js

```diff
@@ -19,7 +19,7 @@ var sourceNameOverride = process.env.SOURCE_NAME_OVERRIDE || 'none';          //
 var encoding = process.env.ENCODING || 'utf-8';  // default is utf-8
 
 // Include logStream and logGroup as json fields within the message. Required for SumoLogic AWS Lambda App
```

Comment:
Is this still true?

Reply:
default is false because most customers do not require this info